lib/utils/compose.js

```diff
@@ -530,6 +530,8 @@ const getPreviousRepos = (sdk, docker, logger, appID) =>
 						return registry.imageName;
 					});
 				});
+			} else {
+				return Promise.resolve([]);
 			}
 		})
 		.catch(e => {
```

Comment:
Wont we get an undefined returned here too?

Reply:
Yes